pkg/cmd/roachtest/tests/activerecord_blocklist.go

```diff
@@ -30,51 +21,76 @@ var activeRecordBlocklists = blocklistsForVersion{
 // in the test log.
-var activeRecordBlockList23_1 = blocklist{}
-
-var activeRecordBlockList22_2 = blocklist{}
-
-var activeRecordBlockList22_1 = blocklist{}
-
-var activeRecordBlockList21_2 = blocklist{}
-
-var activeRecordBlockList21_1 = blocklist{}
-
-var activeRecordBlockList20_2 = blocklist{}
-
-var activeRecordIgnoreList23_1 = activeRecordIgnoreList22_2
-
-var activeRecordIgnoreList22_2 = activeRecordIgnoreList22_1
-
-var activeRecordIgnoreList22_1 = blocklist{
-	"CockroachDB::PostgresqlIntervalTest#test_interval_type":               "flaky",
-	"ConcurrentTransactionTest#test_transaction_isolation__read_committed": "flaky - https://github.com/cockroachdb/activerecord-cockroachdb-adapter/issues/237",
-	"FixturesTest#test_create_fixtures":                                    "flaky - FK constraint violated sometimes when loading all fixture data",
-	"IgnoreFixturesTest#test_ignores_books_fixtures":                       "flaky - FK constraint violated sometimes when loading all fixture data",
-	"IgnoreFixturesTest#test_ignores_parrots_fixtures":                     "flaky - FK constraint violated sometimes when loading all fixture data",
-	"PostgresqlIntervalTest#test_interval_type":                            "flaky",
-}
-
-var activeRecordIgnoreList21_2 = blocklist{
-	"ActiveRecord::CockroachDB::UnloggedTablesTest#test_gracefully_handles_temporary_tables": "modified to pass on 20.2",
-	"CockroachDB::PostgresqlIntervalTest#test_interval_type":                                 "flaky",
-	"ConcurrentTransactionTest#test_transaction_isolation__read_committed":                   "flaky - https://github.com/cockroachdb/activerecord-cockroachdb-adapter/issues/237",
-	"FixturesTest#test_create_fixtures":                                                      "flaky - FK constraint violated sometimes when loading all fixture data",
-	"IgnoreFixturesTest#test_ignores_books_fixtures":                                         "flaky - FK constraint violated sometimes when loading all fixture data",
-	"IgnoreFixturesTest#test_ignores_parrots_fixtures":                                       "flaky - FK constraint violated sometimes when loading all fixture data",
-}
-
-var activeRecordIgnoreList21_1 = blocklist{
-	"ActiveRecord::CockroachDB::UnloggedTablesTest#test_gracefully_handles_temporary_tables": "modified to pass on 20.2",
-	"CockroachDB::PostgresqlIntervalTest#test_interval_type":                                 "flaky",
-	"FixturesTest#test_create_fixtures":                                                      "flaky - FK constraint violated sometimes when loading all fixture data",
-	"IgnoreFixturesTest#test_ignores_books_fixtures":                                         "flaky - FK constraint violated sometimes when loading all fixture data",
-	"IgnoreFixturesTest#test_ignores_parrots_fixtures":                                       "flaky - FK constraint violated sometimes when loading all fixture data",
+var activeRecordBlocklist = blocklist{
```

Comment:
for the ones that have issues, the `value` of the map can just be the issue number

Reply:
Fixed!